docs/contributing/gatsby-docs-translation-guide.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Gatsby Docs Translation Guide
+---
+
+Since the [RFC on localization](https://github.com/gatsbyjs/rfcs/pull/42) for Gatsby, there is now an ongoing effort to translate the content on Gatsbyjs.org into multiple languages. For members of the community around the world building Gatsby sites and learning about web development, having docs and learning materials in your own language provides a lot of value. There's also a great opportunity for folks like you to contribute to the translations of the Gatsby docs.
+
+If you're a native speaker of a language other than English and you're interested in helping translate gatsbyjs.org, here is some information for you to know:
```

Comment:
do we only want to restrict to native speakers? Wouldn't fluency be enough?

Reply:
Ooh, good call. "Fluent speakers" is an easy change.